sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawPod.java

```diff
@@ -19,7 +19,6 @@
     private final String image;
     private final WriteOnceLinkedHashMap<String, RawNetwork> networks;
     private final WriteOnceLinkedHashMap<String, RawRLimit> rlimits;
-    private final String user;
```

Comment:
We have to leave it on for now to retain upgradeability from versions that don't have a service user. I can add a deprecation notice.